app/javascript/components/set-ownership-form/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { http } from '../../http_api';
+import createSchema from './ownership-form.schema';
+
+class SetOwnershipForm extends Component {
+  state = {
+    initialValues: {},
+    userOptions: [],
+    groupOptions: [],
+  };
+
+  componentDidMount() {
+    const { ownershipIds } = this.props;
+    this.loadInitialData(ownershipIds);
+  }
+
+  addOptions = (ownerShipIds, type, data) => {
+    const ret = [];
+    if (ownerShipIds.length > 1) {
+      ret.push([__('Don\'t change'), 'dont-change']);
+    }
+    ret.push([type === 'user' ? __('No Owner') : __('No User Group'), '']);
+
+    return [...ret, ...data];
+  };
+
+  loadInitialData = (ownershipIds) => {
+    Promise.all([
+      http.get('/api/users?expand=resources'),
```

Comment:
For API requests we have a separate library called `API`, you can import it via:
```js
import { API } from '../../http_api';
```